.github/workflows/ci.yaml

```diff
@@ -100,6 +100,10 @@ jobs:
         if: matrix.os == 'ubuntu-20.04'
         run: ./run ormolu:check
 
+      - name: Build deploy TS package
+        if: startsWith(github.ref, 'refs/tags/v') && (matrix.os == 'ubuntu-20.04' || matrix.os == 'macos-latest')
```

Comment:
I guess we can always build (it will add some time). Do you mean just as a safety check in case someone updates the TS but it fails to build or something? We only need it in the data dir though for releases. But we can always do this step as a safety check if you think it's worth it.

Reply:
Aha, ok in that case maybe we don't have to -> what I thought is that possibly some of our tests (e2e?) also need this. But I guess we have none running this stuff right now? Should we have? And yes, even just checking it compiles is something I think.
How much time does it add though? We could try avoiding this extra time by doing Github CI caching if needed, although I wouldn't go into that now to not overcomplicate everything.

Reply:
I removed the `v` check so we can just run it each time, doesn't take too long 👍🏻